models/models_methods_test.go

```diff
@@ -397,7 +473,16 @@ workflows:
 		require.NoError(t, item.Validate())
 	}
 
-	t.Log("it validates pull-request trigger item")
+	t.Log("it validates pull-request trigger item with triggered pipeline")
+	{
+		item := TriggerMapItemModel{
+			PullRequestTargetBranch: "master",
```

Comment:
I'm not familiar with the code base, not sure if it adds value to test different trigger types separately, when the pipeline/workflow existence requirement is same for all.